gateway-js/src/supergraphManagers/UplinkSupergraphManager/__tests__/UplinkSupergraphManager.test.ts

```diff
@@ -0,0 +1,65 @@
+import mockedEnv from 'mocked-env';
+
+import { UplinkSupergraphManager } from '@apollo/gateway';
+
+import { DEFAULT_UPLINK_ENDPOINTS } from '../index';
```

Comment:
In case you're unaware, this is fine too. If what you have is a preference then by all means, please keep it as is.
```suggestion
import { DEFAULT_UPLINK_ENDPOINTS } from '..';
```